wepool-app/src/components/LoginTextInput.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react'
+import { View, SafeAreaView, StyleSheet, Text, TextInput as Input } from 'react-native'
+import Colors from '../core/Colors'
+
+type Props = {
+    label? : string,
+    placeholder?: string,
+    errorText?: string, 
+    description?: string,
+    props?: any
+}
+
+export default function LoginTextInput({label,placeholder, errorText, description, props}: Props) {
```

Comment:
Es mejor usar sintaxis de named export en vez de usar default exports. Lee [este blog](https://www.bundleapps.io/blog/use-named-exports-over-default-exports-in-javascript) para entender por qué. 
Por ejemplo
```typescript
export const LoginTextInput(params: Props) {
     ...
}
```